src/styles/theme/oc-sidebar.scss

```diff
@@ -0,0 +1,112 @@
+.oc-sidebar {
+  display: flex;
+  flex-flow: column wrap;
+  width: $width-medium;
+  height: 100%;
+  padding: $gutter 0;
+  max-height: 100%;
```

Comment:
What do we want to happen when there are too many menu items? I'd add `max-width: $width-medium;` and let the menu overflow happen vertically (you set it to auto already). Still not ideal to have it overflowing at all, but better that then some weird width changes I guess. If you go this way, also set `max-width: 100vw;` for the fixed variant.